server/src/helpers/translation.js

```diff
@@ -47,12 +49,38 @@ const getTranslatedText = async (text, originalLanguage) => {
   }
 }
 
+const readTranslationMockFile = async file => {
+  try {
+    const rawData = fs.readFileSync(file)
+    const data = JSON.parse(rawData)
+    return data
+  } catch (error) {
+    console.error(error)
+  }
+}
+
+const findData = (data, text) => {
+  const keys = Object.keys(data)
+  for (const key of keys) {
+    const item = data[key]
```

Comment:
Object.entries fait ce que tu veux plus efficacement
```suggestion
  for (const [key, item] of Object.entries(data)) {
```